packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/ImageRTL.txt

```diff
@@ -13,16 +13,16 @@
     VerticalAlignment=Stretch
     Visibility=Visible
     Width=500
-  [Microsoft.ReactNative.ViewPanel]
-      Background=[NULL]
-      BorderBrush=#00000000
-      BorderThickness=0,0,0,0
-      Clip=[NULL]
-      CornerRadius=0,0,0,0
-      FlowDirection=LeftToRight
-      HorizontalAlignment=Stretch
-      Margin=0,0,0,0
-      RenderSize=500,300
-      VerticalAlignment=Stretch
-      Visibility=Visible
-    [Windows.UI.Xaml.DependencyObject]
+    [Microsoft.ReactNative.ViewPanel]
+        Background=[NULL]
+        BorderBrush=#00000000
+        BorderThickness=0,0,0,0
+        Clip=[NULL]
+        CornerRadius=0,0,0,0
+        FlowDirection=LeftToRight
```

Comment:
>FlowDirection=LeftToRight [](start = 8, length = 25)

Saw your earlier comment...  LeftToRight is correct, to match the behavior of the XAML Image element, which explicitly does not mirror images in RTL.  Kai made a change recently to ensure this, which explains why this master hasn't changed.